frontend/src/utils/formatDate.js

```diff
@@ -0,0 +1,14 @@
+import dayjs from "dayjs";
+
+// date format for the short year format - to be used in javascript.
+// the date format for database queries is defined in backend/src/models/reports/helpers/index.js
+const DATE_FORMAT_SHORT_YEAR = "DD-MMM-YY";
```

Comment:
Are date values formatted this way in the tables too? If so, how does sorting a date column work since I assume it sorts alphabetically.

Reply:
the date sorting works alphabetically, so that's a new problem, potentially
![image](https://github.com/bcgov/gdx-agreements-tracker/assets/122112933/b218b3e7-c56e-4d99-820a-e7b0946387cd)

Reply:
It looks like there are features of the Data Grid (https://mui.com/x/react-data-grid/column-definition/#value-formatter) we could use to keep the values as date objects so the sorting works using date sorting while still displaying the cell values in the date format we want.

Reply:
This sorting is broken on development branch too. (see https://localhost:3000/projects)

That's a whole other ticket imho.

Reply:
https://gdx-agreements-tracker-test.apps.silver.devops.gov.bc.ca/projects
https://gdx-agreements-tracker-test.apps.silver.devops.gov.bc.ca/contracts

both have broken sorting by date

Reply:
@ShawnTurple do we have a bug ticket in the system for this bug?